src/sfctl/tests/help_text_test.py

```diff
@@ -0,0 +1,364 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Tests to ensure that -h does not return error when run. Also ensures that help
```

Comment:
I feel like we should be request this functionality from the `knack` module. We should probably add an option to make it error a command if it cannot find the help description for an argument or command. That would be easier than manually parsing the output. The output is subject to change depending on how `knack` formats it, so we would have to re-write this test every time it changed, not the right place to test it.

Reply:
You're right. But in the meantime, this gives us some coverage. I'll request the change from knack and remove this test when/if they make the change.